apps/transport/lib/transport/resource.ex

```diff
@@ -30,6 +30,19 @@ defmodule Transport.Resource do
 
   def endpoint, do: Application.get_env(:transport, :gtfs_validator_url) <> "/validate"
 
+  @doc """
+  Is the dataset type corresponding to a public transit file
+  ## Examples
+      iex> Resource.is_transit_file?("rail")
+      true
+      iex> Resource.is_transit_file?("micro-mobility")
+      false
+  """
+  def is_transit_file?(type) do
+    ["public-transit", "long-distance-coach", "rail"]
+    |> Enum.member?(type)
```

Comment:
I thought we don't use pipe if there's just one pipe.

Reply:
I agree, but as it is a long list, it’s less readable i think. Otherwise I could put the list in a constant, but it’s neither convincing IMHO